pyats-intro.py

```diff
@@ -0,0 +1,79 @@
+import sys
+from IPython import embed
+import logging
+import unicon
+
+from genie.conf import Genie
+from ats.topology import loader
+from genie.abstract import Lookup
+from genie.libs import ops # noqa
+
+
+
+
+
+if __name__ == '__main__':
+
+    # local imports
+    import argparse
+
+    parser = argparse.ArgumentParser(description="standalone parser")
+    parser.add_argument('--testbed', default='./default_testbed.yaml',
+                        dest='testbed', type=loader.load)
+    parser.add_argument('--device', dest='device_name')
+
+    args, unknown = parser.parse_known_args()
+
+    device_name = args.device_name
+
+    # pyats testbed != genie testbed
+    genie_testbed = Genie.init(args.testbed)
+
+    # this gives us device_name as Device Object e.g dist1
+    vars()[device_name] = genie_testbed.devices[device_name]
```

Comment:
the more appropriate way would be to do locals()

but then again - device_name could have a dash, then it's an inappropriate variable name.

i would suggest to not show this part. it will definitely confuse novice users